admin_manual/configuration_server/occ_command.rst

```diff
@@ -685,8 +686,6 @@ Apply correct permissions to your ownCloud directories; see
 :ref:`strong_perms_label`. Then choose your ``occ`` options. This lists your 
 available options::
 
-To be able to run the installation command, you have to ``cd`` into the root folder of ownCloud. Using path-to-owncloud won't work.
```

Comment:
Yes, it is intended. Since it's fundamental to running occ it belongs somewhere else on the page. Probably several places as a reminder. It's on my list for today.